component/src/main/java/org/wso2/carbon/identity/authenticator/bitly/BitlyAuthenticator.java

```diff
@@ -263,4 +273,43 @@ protected Map<ClaimMapping, String> getSubjectAttributes(OAuthClientResponse tok
         }
         return claims;
     }
+
+    @Override
+    protected String sendRequest(String url, String accessToken) throws IOException {
```

Comment:
Does this method is used to invoke anything other than the user info endpoint ?

Reply:
If so, can we consider adding the authenticator identifier to the debug logs ?

ex: `log.debug("Bitly user info response: " + builder.toString());`

Reply:
This method is used in bitly authenticator only to invoke user info endpoint only. We can modify the debug log

Reply:
Fixed